src/components/Typography/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { TypographyProps } from './TypographyProps';
+
+const Typography = ({
+    font,
+    size,
+    color,
+    textLabel,
```

Comment:
define default value for this.. because you declare is as optional...

```suggestion
    font = "default font",
    size = "large",
    color ="green",
    textLabel,
```

but for textLabel.. you can keep it empty.. cause it is expected to be empty in some case.. 

for color, i think you need to have the default value right?